src/core/state/hiddenAssets/hiddenAssets.ts

```diff
@@ -0,0 +1,43 @@
+import create from 'zustand';
+
+import { createStore } from '../internal/createStore';
+
+type UpdateHiddenAssetArgs = {
+  uniqueId: string;
+};
+
+type UpdateHiddenAssetFn = ({ uniqueId }: UpdateHiddenAssetArgs) => void;
+
+export interface HiddenAssetState {
+  hiddenAssets: string[];
+  addHiddenAsset: UpdateHiddenAssetFn;
+  removeHiddenAsset: UpdateHiddenAssetFn;
+}
+
+export const hiddenAssets = createStore<HiddenAssetState>(
```

Comment:
`hiddenAssets` should be `hiddenAssetsStore` here

Reply:
If we needed a hidden assets hook at some point (like to display a list of just the hidden assets) we wouldn't want the name collision